docs/manuals/measurement/readout_mitigation.rst

```diff
@@ -118,13 +109,9 @@ Mitigation example
         qc.cx(i - 1, i)
     qc.measure_all()
 
-.. jupyter-execute::
```

Comment:
I also merged adjacent `jupyter-execute` cells because they don't work well. In the rendered docs, they look like one big cell any way, but they still have individual buttons for copying the code to the clipboard, so you think you are copying the whole merged cell but you only get a subsection of it.